syft/formats/syftjson/model/package_test.go

```diff
@@ -8,9 +8,11 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/anchore/syft/syft/license"
 	"github.com/anchore/syft/syft/pkg"
 )
 
+// TODO: add new license struct to this test
```

Comment:
this looks like a TODO that was meant to be done before merging this PR (not a TODO for a follow up PR, but I could be missing something).